2413.smallest-even-multiple.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * URL of this problem
+ * https://leetcode.com/problems/smallest-even-multiple/
+ */
+
+/**
+ * @param {number} n
+ * @return {number}
+ */
+var smallestEvenMultiple = function(n) {
+    if (n % 2 !== 0) {     
+        return n * 2;      
+    } else {        
+        return n;
+    }
```

Comment:
```suggestion
return n % 2 === 0 ? n : n * 2; 
```
You can also do this.

Reply:
sounds good @HidemichiShimura